RELEASING.md

```diff
@@ -0,0 +1,186 @@
+<!-- markdownlint-configure-file { "MD013": { "line_length": 200 } } -->
+# Releasing
+
+## Overview
+
+This repo contains three types of release artifacts:
+
+* **Go Binaries** for [Terraform Engine](./docs/tfengine), [Policy Generator](./docs/policygen), and [Terraform Importer](./docs/tfimport).
```

Comment:
Optional: consider including the tags here so it is prominent right at the top

* Go Binaries (tag: `vX.Y.Z`)
*  Templates (tag: `templates-vX.Y.Z`)

Reply:
Done.